modules/apps/29-fee/keeper/genesis_test.go

```diff
@@ -0,0 +1,103 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	transfertypes "github.com/cosmos/ibc-go/modules/apps/transfer/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
+)
+
+func (suite *KeeperTestSuite) TestInitGenesis() {
+	suite.SetupTest()
+
+	// build PacketId & Fee
+	refundAcc := suite.chainA.SenderAccount.GetAddress()
+	ackFee := validCoins
+	receiveFee := validCoins2
+	timeoutFee := validCoins3
+	packetId := &channeltypes.PacketId{ChannelId: ibctesting.FirstChannelID, PortId: types.PortID, Sequence: uint64(1)}
```

Comment:
Ah I didn't realize we added it. `NewPacketId` should go in the channel types not in fee types. We should remember to remove it later on. It should be a PR to main for I think `NewPacketId` (I mentioned this on the other PR I think)

Reply:
I updated but I added a TODO to remove the helper & also updated the fn def so that it has the correct params. It was hardcoded to take the fee portID (required updating the queries also). 

https://github.com/cosmos/ibc-go/pull/557/commits/0421fd9cc9e834a4ff1ca7a102c9c5459454e394